src/javafundamentals/streams/Ex06MaybeInThere.java

```diff
@@ -8,4 +12,10 @@
  */
 
 public class Ex06MaybeInThere {
+    public static void main(String[] args) {
+        IntStream randomStream = new Random().ints(10, 1, 10);
+        OptionalInt any = randomStream.findAny();
+        any.ifPresent(i-> System.out.println(i));
+        any.orElse(0);
```

Comment:
here you are just using the optional `any` to use its method `orElse()` . this method returns a number if it is present or else returns 0. 
You are not interacting with this value at all.